examples/05_stable_diffusion/scripts/compile.py

```diff
@@ -44,6 +45,21 @@ def compile_diffusers(
 ):
     logging.getLogger().setLevel(logging.INFO)
     torch.manual_seed(4896)
+    
+    """
+        Set the OS environment variable AITEMPLATE_WORK_DIR to point to an absolute path to a directory which 
+        will be used to save the AIT compiled model artifacts. Make sure the OS user running this script has read and write 
+        permissions to this directory. By default, the artifacts will be saved under tmp/ folder of the 
+        current working directory. 
+    """
+    
+    env_name = "AITEMPLATE_WORK_DIR"
+    try:
+        if os.environ[env_name]:
+            local_dir = os.path.join(os.environ[env_name], 'diffusers-pipeline', 'stabilityai','stable-diffusion-v2')
+            print("The value of", env_name, " is ", local_dir)
+    except KeyError:
+        print("AITEMPLATE_WORK_DIR environment variable is not set. Using default local dir as ",local_dir)
```

Comment:
I've noticed that this code is duplicated across a bunch of files. Can it be deduped and put into a helper function instead?

Reply:
I thought about it however, I did not pursue to decouple it into a seperate function is due to the fact, some code expects to load the models from specific folder path like "diffusers-pipeline/stabilityai.." vs other code expects just the parent directory with "tmp/". Thus, in order to keep the existing code intact and retain backward compatibility, I chose this approach.

Reply:
We can work on changing this and decoupling it as a second iteration on top of this PR. Right now, as it was a burning issue for us and was a blocker, had to push this PR.